lms/djangoapps/course_home_api/course_metadata/v1/serializers.py

```diff
@@ -33,15 +33,17 @@ class CourseHomeMetadataSerializer(VerifiedModeSerializerMixin, serializers.Seri
     """
     Serializer for the Course Home Course Metadata
     """
+    can_load_courseware = serializers.BooleanField()
```

Comment:
Nit: I'd argue this variable should be renamed to better capture that this is only handling if the MFE is active and not if the learner can load _any_ courseware (MFE or otherwise).

Reply:
We talked about this a little off-PR. I don't disagree, but this is a historical name that would be extra work to rename, and I didn't want to bother right now. So nit acknowledged and agreed with, but I'm being a bit lazy.